tests/src/test/330/scala/com/nvidia/spark/rapids/IntervalSuite.scala

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids
+
+import java.io.File
+import java.time.{Duration, Period}
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+/**
+ * Can not put this suite to cache_test.py because of currently Pyspark not have year-month type.
+ * See: https://github.com/apache/spark/blob/branch-3.3/python/pyspark/sql/types.py
+ */
+class IntervalSuite extends SparkQueryCompareTestSuite {
+
+  /**
+   * Set environment variable: "spark.sql.cache.serializer"
+   * "spark.sql.cache.serializer" is a static SQL configuration, can NOT set/unset it.
+   * Should specify it as environment variable when running the MVN, e.g:
+   * SPARK_CONF="spark.sql.cache.serializer=com.nvidia.spark.ParquetCachedBatchSerializer" mvn test
+   * This method is a hack code to set environment variable in the test code. it's a workaround.
+   * Should move this test suite to cache_test.py after Pyspark supports year-month type
+   */
+  private def setPCBS() = {
```

Comment:
This is not recommended as you may know. Instead, [you can control the spark session creation by setting this static config or not here](https://github.com/NVIDIA/spark-rapids/blob/branch-22.04/tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala#L68).

Reply:
This is temp and will be removed after Pyspark supports year month interval, so i would suggest not change the spark session creation for this.